libkbfs/coalescing_context.go

```diff
@@ -55,33 +62,41 @@ func (ctx *CoalescingContext) addContextLocked(other context.Context) {
 // canceled to avoid a goroutine leak.
 func NewCoalescingContext(parent context.Context) (*CoalescingContext, context.CancelFunc) {
 	ctx := &CoalescingContext{
-		Context:  context.Background(),
-		closeCh:  make(chan struct{}),
+		// Make the parent's `Value()` method available to consumers of this
+		// context.
```

Comment:
Might be worth adding something like "For example, this maintains the parent log debug tags for all future uses of this context.".